sky/resources.py

```diff
@@ -267,6 +277,15 @@ def disk_size(self) -> int:
     def image_id(self) -> Optional[Dict[str, str]]:
         return self._image_id
 
+    @property
+    def disk_type(self) -> str:
+        return self._disk_type
+
+    def get_cloud_disk_type(self) -> str:
+        # default to low if not specified
+        disk_type = 'low' if self.disk_type is None else self.disk_type
+        return self.cloud.get_disk_type(disk_type)
```

Comment:
Showing the disk_type specifed in the yaml `low`, `medium`, `high` is good enough. We should not show the details.